ui/components/ui/toggle-button/toggle-button.component.js

```diff
@@ -52,7 +52,7 @@ const ToggleButton = (props) => {
   const modifier = value ? 'on' : 'off';
 
   return (
-    <div
+    <label
```

Comment:
It looks like the cursor changes to `pointer` when you hover over the toggle. What do you think about adding `cursor: pointer` to `toggle-button` so that the cursor changes when you hover over the text too?